src/airportDetailsRetriever.js

```diff
@@ -0,0 +1,28 @@
+const airportDetails = require('../resources/airportsInfo.json');
+
+const getAirportInformation = (code) => {
+  if (typeof code !== 'string' || code.length !== 3) throw Error('Invalid, enter a proper airport code!');
+  let foundFlag = 0;
+  let airportInformation;
+  airportDetails.forEach((airport) => {
```

Comment:
This is fine, but try `find` or `filter`. It makes code more easily understandable